docs/network-iot/run-an-lns/fund-an-oui.mdx

```diff
@@ -0,0 +1,94 @@
+
+## Estimating Usage
+
+<figure className="screensnippet-wrapper">
+  <iframe
+    src="https://dune.com/embeds/2608923/4325514"
+    frameborder="0"
+    width="100%"
+    height="350px"
+  ></iframe>
+  <figcaption>
+    Up-to-date usage metrics for all OUIs on the Helium network. <br /> To get a sense of traffic
+    and DC usage on the network, check out the full{' '}
+    <a href="https://dune.com/helium-foundation/helium-data-credits">Dune Data Credit Dashboard</a>.
+  </figcaption>
+</figure>
+<br />
+
+Data credits are charged per 1DC per 24b increment for each response of an uplink payload. For a
+helpful visualization of DC usage of devices for a given period, check out the
+[Data Credit calculator](/tokens/data-credit#data-transfer-calculator) on the DC documentation.
+
+Only uplinks are charged on Helium. Downlinks and join requests are not billed.
+
+### Understanding Multibuy (`max_copies`)
+
+The Helium Network charges for each response for a given payload. This is known as "multibuy" or
+`max_copies`. Each Hotspot that reports a payload from a given device is eligible to be rewarded.
+The LNS operator has a choice of how many reports they would like to buy.
+
+Multibuy sets the maximum number of Hotspots that can deliver their receipt of a message. For
+instance, for a multibuy value of 10 on a device that only receives 8 reports, the OUI would be
+charged 12 DC ($0.00008 USD). This assumes the payload was less than 24b.
+
+Multibuy is helpful in several scenarios where knowing a wide selection of available Hotspots is
+valuable:
+
+- **Ensuring successful downlinks.**  
+  Helium reports Hotspots responses in chronological order. This can be challenging in scheduling
+  downlinks because the first Hotspot to report may not necessarily be the closest or have the
+  strongest RSSI/SNR attributes. This is important for downlink scenarios such as Class-C devices,
+  ADR, confirmed uplinks, or standard downlinks on Class-A devices. Leveraging multibuy is an easy
+  way to ensure that there is a strong Hotspot available to deliver the downlink message.
+- **Multilateration.**  
+  For devices that do not have internal location-solving abilities (GPS), the metadata of the packet
+  report and Helium Network can be leveraged to solve for a device location.
+
+## Tracking OUI Balance
+
+The balance of an OUI is held in an escrow account that is generated at OUI creation. This escrow
+account is what allows the Helium Network to appropriately deduct DC as network transfer occurs.
+
+The escrow account is the main source for understanding the remaining balance of an OUI.
+
+When monitoring the OUI balance, it is worth noting that the minimum usage deducted at a time is
+10,000 DC[^1].
+
+[^1]:
+    `BURN_THRESHOLD` on
+    [GitHub](https://github.com/helium/oracles/blob/7c9f5b4e6f8e47be80eaf3c6cb4cc703373246ef/iot_packet_verifier/src/pending_burns.rs#L32)
+
+- Escrow Account
+  - How to find
```

Comment:
@ChewingGlass Are there any quick notes we can provide here to help folks resolve their escrow account (other than using the Sphere DC Portal)

Reply:
This is docs, you could insert a docs widget. Just have a text input and feed it to `escrowAccountKey(delegatedDataCreditsKey(subDaoKey(IOT_MINT)[0], <Router Key>)[0])[0]`

It's PDAs all the way down.

Reply:
If you view the resulting account on solana explorer you should see the balance